### DISCUSSION TITLE
Player-⭐ed list

### DISCUSSION THREAD

**Discussion by KeifCheif** [DiscordDungeons/Bugs#1011] [Ideas]
Give players the ability to make their own starred list of items that they see as more important or special, that they can access using #!starredinv or #!invstar.

**Comment by FelixArgyle** [comment]
Maybe not a whole new inv cmd just incorporating it in current inv would be better
i.e  starred items show up first in your inventory (even with a invsort applied)
something like
```diff
!===== [Felix Argyle's Inventory (Page 1/17)] =====!
+ 1,442,252,538 Gold.
+ 864,799 x Rocks [⭐]
+ 599,300 x Health Potions
+ 506,962 x Logs
+ 419,028 x Dust
+ 385,589 x Immortal King's Sacraments
+ 216,613 x Felix's Orbs
+ 206,657 x Sugar Beets
+ 194,660 x Olives
+ 161,602 x Maple Logs
+ 151,741 x Iron Ores
+ 145,036 x Rock Salt
+ 134,929 x Raven Feathers
+ 113,085 x Golden Feathers
+ 92,266 x Enhancing Stones
!==================================================!
```

**Reply by KeifCheif** [reply]
Yeah that'd probably be easier, just something to give the item priority over others so they show up first in the inventory, or something that differentiates them from others so they're easier to find
